charmhelpers/contrib/storage/linux/utils.py

```diff
@@ -132,8 +128,6 @@ def mkfs_xfs(device, force=False, inode_size=None):
     if inode_size:
         if inode_size >= 256 and inode_size <= 2048:
             cmd += ['-i', "size={}".format(inode_size)]
-        elif inode_size != 0:
-            log("Config value xfs-inode-size={} is invalid. Using system default.".format(inode_size), level=WARNING)
 
```

Comment:
Hey, I think I wasn't clear in my sentence. We need to remove the check, but leave the warn so the user knows that he is using a bad value:

```python
if inode_size >= 256 and inode_size <= 2048:
    cmd += ['-i', "size={}".format(inode_size)]
else:
    log("Config value xfs-inode-size={} is invalid. Using system default.".format(inode_size), level=WARNING)
```